### ISSUE TITLE
symdeps throws error when creating (sym)links on Windows OS

### ISSUE THREAD

**Issue by Boegie** [agarzola/symdeps#52]
Using this fine package on Windows (11 to be exact) to build the chromatic Drupal custom theme throws an error when used in `composer robo theme:build default` at the stage of trying to create (sym)links.

Stack trace:

```
$ composer robo theme:build default
> Composer\Config::disableProcessTimeout
> robo --ansi theme:build default

theme build
===========

building theme at web/themes/chromatic
--------------------------------------

 [Exec] Running yarn build in web/themes/chromatic
yarn run v1.22.17
$ yarn calliope build
$ yarn install && calliope build
[1/4] Resolving packages...
success Already up-to-date.
$ symdeps
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
internal/fs/utils.js:332
    throw err;
    ^

Error: ENOENT: no such file or directory, link 'node_modules\hammerjs\hammer.min.js' -> 'D:\htdocs\chromatichq.com\web\themes\chromatic\build\vendor\hammerjs\hammer.min.js'
    at Object.linkSync (fs.js:1239:3)
    at D:\htdocs\chromatichq.com\web\themes\chromatic\node_modules\symdeps\lib\symdeps.js:45:20
    at Array.forEach (<anonymous>)
    at D:\htdocs\chromatichq.com\web\themes\chromatic\node_modules\symdeps\lib\symdeps.js:15:16
    at Array.forEach (<anonymous>)
    at module.exports (D:\htdocs\chromatichq.com\web\themes\chromatic\node_modules\symdeps\lib\symdeps.js:10:29)
    at module.exports (D:\htdocs\chromatichq.com\web\themes\chromatic\node_modules\symdeps\index.js:33:3)
    at Object.<anonymous> (D:\htdocs\chromatichq.com\web\themes\chromatic\node_modules\symdeps\bin\cli:2:15)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10) {
  errno: -4058,
  syscall: 'link',
  code: 'ENOENT',
  path: 'node_modules\\hammerjs\\hammer.min.js',
  dest: 'D:\\htdocs\\chromatichq.com\\web\\themes\\chromatic\\build\\vendor\\hammerjs\\hammer.min.js'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
error Command failed with exit code 1.
 [Exec]  Exit code 1  Time 6.612s
 [notice] Stopping on fail. Exiting....
 [error]  Exit Code: 1
 [error]    in task Robo\Task\Base\Exec

  yarn run v1.22.17
$ yarn calliope build
$ yarn install && calliope build
[1/4] Resolving packages...
success Already up-to-date.
$ symdeps
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Script robo --ansi handling the robo event returned with error code 1
```

**Comment by Boegie** [comment]
PR incoming

**Comment by agarzola** [comment]
Fixed in #53.
